platform/ios/src/MGLMapView.mm

```diff
@@ -518,6 +523,11 @@ - (void)commonInit
     _annotationViewReuseQueueByIdentifier = [NSMutableDictionary dictionary];
     _selectedAnnotationTag = MGLAnnotationTagNotFound;
     _annotationsNearbyLastTap = {};
+    
+    // TODO: This warning should be removed when automaticallyAdjustsScrollViewInsets is removed from
+    // the UIViewController api.
+    NSLog(@"%@ WARNING UIViewController.automaticallyAdjustsScrollViewInsets is deprecated use MGLMapView.automaticallyAdjustContentInset instead.",
```

Comment:
Use `MGLLogInfo` here?

Reply:
We want the warning logged into the console directly using our logging framework can't warranty that.